test/integration_test.exs

```diff
@@ -2,39 +2,78 @@ defmodule ConfigCat.IntegrationTest do
   # Must be async: false to avoid a collision with other tests.
   # Now that we only allow a single ConfigCat instance to use the same SDK key,
   # one of the async tests would fail due to the existing running instance.
-  use ExUnit.Case, async: false
+  use ConfigCat.Case, async: false
 
   alias ConfigCat.Cache
   alias ConfigCat.CachePolicy
   alias ConfigCat.InMemoryCache
+  alias ConfigCat.LocalMapDataSource
 
-  @sdk_key "PKDVCLf-Hq-h-kCzMp-L7Q/PaDVCFk9EpmD6sLpGLltTA"
+  @sdk_key "configcat-sdk-1/PKDVCLf-Hq-h-kCzMp-L7Q/1cGEJXUwYUGZCBOL-E2sOw"
 
-  test "raises error if SDK key is missing" do
-    nil
-    |> start_config_cat()
-    |> assert_sdk_key_required()
-  end
+  describe "SDK key validation" do
+    test "raises error if SDK key is missing" do
+      nil
+      |> start()
+      |> assert_sdk_key_required()
+    end
 
-  test "raises error if SDK key is an empty string" do
-    ""
-    |> start_config_cat()
-    |> assert_sdk_key_required()
-  end
+    test "raises error if SDK key is an empty string" do
+      ""
+      |> start()
+      |> assert_sdk_key_required()
+    end
 
-  @tag capture_log: true
-  test "raises error when starting another instance with the same SDK key" do
-    {:ok, _} = start_config_cat(@sdk_key, name: :original)
+    for sdk_key <- [
+          "key",
+          "configcat-proxy/key",
+          "1234567890abcdefghijkl01234567890abcdefghijkl",
+          "configcat-sdk-2/1234567890abcdefghijkl/1234567890abcdefghijkl",
+          "configcat/1234567890abcdefghijkl/1234567890abcdefghijkl"
+        ] do
```

Comment:
Again, the Python SDK doesn't have all of these tests. You might want to add them.

Reply:
Added in bfa583e

Reply:
Probably this happened because the Python tests were specified earlier and we forgot to update them.

@kp-cat However, I think the JS SDK's test suite is more comprehensive, so it'd be nice to have these tests in Python as well.